config/locales/cy/errors.yml

```diff
@@ -3,7 +3,7 @@ cy:
     errors:
       messages:
         name: nid yw'n enw dilys
-        blank: 'yn ofynnol '
+        blank: 'yn ofynnol'
```

Comment:
@alan did you change this in transifex? Otherwise it will just come back next time I pull from there.